dotnet-logging/LogLevelsWithSerilog/Tests/HomeControllerTests.cs

```diff
@@ -0,0 +1,112 @@
+using AutoFixture;
+using LogLevelsWithSerilog.Controllers;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace LogLevelsWithSerilogTests;
+
+[TestClass]
+public class LogLevelsWithSerilogTests
+{
+    private HomeController? _homeController;
+    private Mock<ILogger<HomeController>>? _loggerMock;
+    private IFixture? _fixture;
+
+    [TestInitialize]
+    public void Setup()
+    {
+        _fixture = new Fixture();
+        _loggerMock = new Mock<ILogger<HomeController>>();
+        _homeController = new HomeController(_loggerMock.Object);
+    }
+
+    [TestMethod]
+    public void WhenLogRouteCalledWithDebug_ThenDebugLogMustBeCalled()
+    {
+        var message = _fixture.Create<string>();
+        var logType = (int)LogLevel.Debug;
```

Comment:
The message part was not the piece I was talking about, but making it a member variable seems okay. I will look at it as a whole and get back to you. It was the logType that should be a local constant. I would suspect your IDE would also show a squiggle or something like that suggesting that it be made into a local constant. I saw the suggestion in both VS2022 and in Rider.

Reply:
I generated a private "TestGenerateLog()" method because testing scenario was similar for the test methods. For each of my test methods, I called TestGenerateLog() with appropriate log level to run test.

Reply:
For each of my test methods, I am testing with a different LogLevel. Thus, implementing them as constants may yield more unreadable code. Currently, I am sending them as enum values to my "TestGenerateLog()" method. I think this is more readable code.

Reply:
@asetil - I am not sure how it would be less readable, I am speaking about local constants, in the method, basically changing from `var logType` to `const LogLevel logType = `

Reply:
@jshergal, actually i did'nt recognize the "local constant" term in your second comment. I thought that i must create private constants upper in the test class. 

After my last changes, there is no "var log = ..." variables anymore, i just sending related Serilog.Events.LogEventLevel enum to TestGenerateLog() method for testing. However, if you think, now the code is more unreadable, i can initialize local constants and send them to TestGenerateLog().